source/slang/ir.cpp

```diff
@@ -3063,6 +3063,14 @@ namespace Slang
                 }
                 break;
 
+            case kIRDecorationOp_LoopControl:
+                {
+                    auto originalDecoration = (IRLoopControlDecoration*)dd;
+                    auto newDecoration = context->builder->addDecoration<IRLoopControlDecoration>(clonedValue);
```

Comment:
Wait, I thought I added this change already. I wonder if these diffs are getting screwed up somehow.